### PR TITLE
Make the buttons in a responsive dialog be responsive.

### PR DESCRIPTION
**Pull request by ssancho14**
* Buttons get wider after the 225.dp breakpoint
* Added dialog activity as a sample to try different dialogs

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt [hunk 1]
```diff
@@ -128,18 +142,42 @@ public fun ResponsiveDialogContent(
                             verticalAlignment = Alignment.CenterVertically,
                         ) {
                             onCancel?.let {
-                                Button(
-                                    imageVector = Icons.Default.Close,
-                                    contentDescription = cancelButtonContentDescription,
+                                androidx.wear.compose.material.Chip(
```
Comment (yschimke) [inline compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt:145]: Can we refactor this into a private ResponsiveButton and keep this Dialog code simpler?

File: compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt [hunk 2]
```diff
@@ -115,6 +122,13 @@ public fun ResponsiveDialogContent(
                 }
                 if (onOk != null || onCancel != null) {
                     item {
+                        val width = LocalConfiguration.current.screenWidthDp
+                        val buttonWidth = if (width < 225 || onOk == null || onCancel == null) {
```
Comment (BowerSteve) [inline compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt:126]: Add a comment that single buttons are not intended to be responsive?

File: compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt (hunk 1 above)
Comment (ssancho14) [reply compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt]: Done

File: compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt (hunk 2 above)
Comment (ssancho14) [reply compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt:132]: Done

**Pull request merged**